Author: Jasper Grimm <jasper.grimm@york.ac.uk>
Description: Use logical not `~` operator instead of `-` to be compatible with
 recent numpy versions
------------------------------------------------------------------------------
diff -ruN charade-elsa-d50cfb060eba.orig/lsa/lsalib.py charade-elsa-d50cfb060eba.p1/lsa/lsalib.py
--- charade-elsa-d50cfb060eba.orig/lsa/lsalib.py	2022-07-12 16:34:27.000000000 +0100
+++ charade-elsa-d50cfb060eba.p1/lsa/lsalib.py	2022-07-12 16:34:58.000000000 +0100
@@ -700,7 +700,7 @@
     #print "V=", V
     #print "V.mask=", V.mask
   nans = (np.nonzero(V.mask)[0]).tolist()      #record indecies
-  V = V[-V.mask]                               #remove nan's
+  V = V[~V.mask]                               #remove nan's
   #print "V=", V
 
   v_num = {}  #values and counts
